src/mockServer/msw/dexie-db.ts

```diff
@@ -27,6 +27,8 @@ export class AppMocks extends Dexie {
 
   appAction!: Dexie.Table<AppAction, string>;
 
+  uploadedFiles!: Dexie.Table<MockUploadedFile, string>;
```

Comment:
For this I think it would be better to create a migration, so that existing databases can be correctly migrated. Not strictly an issue, but I think it would be a good practice.